exporter/exporterhelper/common.go

```diff
@@ -25,7 +25,7 @@ import (
 	"go.opentelemetry.io/collector/config/configmodels"
 )
 
-// ComponentSettings for timeout. The timeout applies to individual attempts to send data to the backend.
+// Settings for timeout. The timeout applies to individual attempts to send data to the backend.
```

Comment:
`go lint` has yelled at me enough times I've internalized that godoc should start with `TimeoutSettings`, though not sure how to reword this doing so and it seems that this already was deviating from that guidance. v0v

Reply:
Not sure what we missed in the config that this is not picked by lint. Will look into this

Reply:
Also was done unintentionally by the IDE because I renamed ComponentSettings -> Settings initially and text was also replaced

Reply:
Apparently it was disabled, starting to enable bunch of these https://github.com/open-telemetry/opentelemetry-collector/pull/2786 will take a bit to fix all errors.